GoogleForCreators/web-stories-wp
# Redesign: Filter button form component

## Context
Adds filter button to the editors form components.


## Relevant Technical Choices

Did not add this to the design-panel since it seems to be very editor-specific and use functionality from the editor (e.g. patterns)


## To-do



## User-facing changes
| Before | After |
|--------|------|
| <img width="301" alt="Screenshot 2021-03-25 at 16 12 27" src="https://user-images.githubusercontent.com/3294597/112486880-94646280-8d7c-11eb-969a-182556e5e5b3.png"> | <img width="305" alt="Screenshot 2021-03-25 at 16 09 33" src="https://user-images.githubusercontent.com/3294597/112486536-40597e00-8d7c-11eb-8915-ae2d5790bb99.png"> |
| <img width="296" alt="Screenshot 2021-03-25 at 16 12 19" src="https://user-images.githubusercontent.com/3294597/112486901-98908000-8d7c-11eb-976b-64c9f7aa741a.png"> | <img width="306" alt="Screenshot 2021-03-25 at 16 09 21" src="https://user-images.githubusercontent.com/3294597/112486546-43546e80-8d7c-11eb-9d74-e0883e07add2.png"> |


### QA






This PR can be tested by following these steps:

See [Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3389%3A64503)

1. Add background media (preferably light, to notice the difference better)
2. Verify the Filters panel looks as in Figma but using the actual image of the selected element.
3. Verify the selected filter displays a light outline
4. Verify that when tabbing through the filters the focus line is displayed per Figma.

### UAT







This PR can be tested by following these steps:

1.
## Checklist




---



Fixes #6499 
Fixes #6527
